conductor/lib/common.py

```diff
@@ -440,14 +444,20 @@ def validate_api_key(config):
         Returns: None
 
         """
-        if 'api_key_path' not in config:
-            config['api_key_path'] = os.path.join(base_dir(), 'auth', 'conductor_api_key')
-        api_key_path = config['api_key_path']
+        # If the key isn't defined, than check for a path with the key
+        if 'api_key' in config:
+            logger.debug("'api_key' is already defined. Ignoring 'api_key_path'")
+            return
+
+        if 'api_key_path' not in config:          
+            api_key_path = Config.get_default_api_key_path()
```

Comment:
on a related note (and in conflict with my suggestion), I'm wondering if we shouldn't have a default api key path at all.  
One of the things that has been very confusing for users (including myself) is when they think they've set  a path to their api key file, but in reality it's the wrong path (or perhaps the key doesn't actually exist on disk).  With the current behavior, the code silently ignores the fact that the specified api key file is missing, and simply falls back to the browser login behavior.  Instead, I think it would be more helpful to raise an exception if the file is not found on disk.  Otherwise, users oftentimes will login via browser, and not realize that their api key isn't actually being used .  This ultimately bites them later.   This isn't something to address in this PR, but wanted to note here. Let's chat IRL.